langchain/callbacks/tracers/langchain.py

```diff
@@ -138,6 +138,7 @@ def _convert_run(run: Union[LLMRun, ChainRun, ToolRun]) -> Run:
             session_id=run.session_id,
             run_type=run_type,
             parent_run_id=run.parent_uuid,
+            example_id=run.example_id,
```

Comment:
We don't use this now, but the proper field is `reference_example_id` on the `Run` schema

Reply:
Got it so I'll change to `reference_example_id=run.example_id,`